yb-voyager/src/tgtdb/event.go

```diff
@@ -89,15 +90,25 @@ func (e *Event) GetParams() []interface{} {
 	}
 }
 
+func (event *Event) GetPreparedStmtName(targetSchema string) string {
+	var ps strings.Builder
+	ps.WriteString(event.getTableName(targetSchema))
```

Comment:
I would suggest collect all the strings in a slice and then use `strings.Join()`. The Join() function takes care of allocating big enough buffer for the `strings.Builder` to avoid memory reallocations.